shared/checks/oval/network_ipv6_static_address.xml

```diff
@@ -4,13 +4,18 @@
     <metadata>
       <title>Manually Assign Global IPv6 Address</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>Manually configure addresses for IPv6</description>
     </metadata>
     <criteria operator="OR">
+      {{%- if product == "rhel6" -%}}
       <extend_definition comment="IPv6 disabled or..."
       definition_ref="sysctl_kernel_ipv6_disable" />
+      {{%- else -%}}
+      <extend_definition comment="IPv6 disabled or..."
+      definition_ref="kernel_module_ipv6_option_disabled" />
+      {{%- endif -%}}
```

Comment:
This if blocks are swaped, `kernel_module_ipv6_option_disabled` should be for rhel6.

Reply:
good catch, fixed